ports/mimxrt10xx/Makefile

```diff
@@ -93,6 +93,9 @@ endif
 
 CFLAGS += $(INC) -Wall -Wno-cast-align -std=gnu11 -nostdlib $(BASE_CFLAGS) $(CFLAGS_MOD) $(COPT)
 
+# Disable some warnings, as do most ports. NXP SDK causes undef, tinyusb causes cast-align
+CFLAGS += -Wno-undef -Wno-cast-align
```

Comment:
Please don't do this. Turning them off applies globally and we want to verify this in most places.

Reply:
This should have been inserted with -Werror, which I forgot to include. But both the NRF and STM32 ports also need to do this because their SDKs invoke these warnings all the time. Would you prefer that the -Werror flag not be set until a later date?

Reply:
Ya, let's do it separately. Thanks!